crates/telio-model/src/api_config.rs

```diff
@@ -187,7 +187,7 @@ where
         Err(_) => return Ok(None),
     };
 
-    let eps: HashSet<_> = eps
+    let eps: Vec<_> = eps
```

Comment:
This deserializer block might actually be possible to remove at some point, since we're no longer getting and deserializing strings for domain objects. I added deduplication of endpoint providers to the constructor of `Device`

Reply:
No longer relevant